blocks/logo2/__icon/logo2__icon.css

```diff
@@ -0,0 +1,20 @@
+.logo2__icon{
+  height: 40px;
+  transition: all 1.0s ease-in;
```

Comment:
@ElenaPodv Continuing with the logo, we typically apply hover and active effects to **interactive elements** such as buttons or links to show they are clickable or indicate an interaction. In our case, it is a **link** that is an interactive element, not an icon, so `:hover` and `:active` effects should be specified for the element with a class `logo2`. 

_You can think of it this way: if, at some point, we removed the `<a>` tag from the HTML, leaving only the decorative icon, it wouldn't be an interactive element any longer, and we wouldn't need hover and active effects on it._

Let's move the `:hover` and `:active` states along with a `transition` property to the link element with the class `logo2`.

Reply:
Done

Reply:
@ElenaPodv thanks a lot for addressing my comment! Please, also delete transition assignment on line 4 🙏

Reply:
Done